src/routes/init/db/index.ts

```diff
@@ -68,15 +68,15 @@ export async function importDbEntries(): Promise<IClaim[]> {
     newClaim.shortCode = getClaimShortCode(newClaim.subject, newClaim.claimNumber, newClaim.grades);
     if (!newClaim.title.includes('Performance')) {
       newClaim.description = getClaimDesc(newClaim.subject, newClaim.shortCode, ELASpec, MATHSpec);
-    if (newClaim.subject !== Subject.MATH && !newClaim.title.includes('Performace')) {
-      newClaim.domain  = [];
-      newClaim.domain.push({
-        title: getClaimDomain(newClaim.subject, newClaim.shortCode, ELASpec),
-      });
+      if (newClaim.subject !== Subject.MATH && !newClaim.title.includes('Performace')) {
```

Comment:
Is this supposed to be spelled `'Performace'`?

Reply:
Yeah I assume so. I'll change it.